sharding-jdbc/sharding-jdbc-core/src/test/java/org/apache/shardingsphere/shardingjdbc/common/base/AbstractEncryptJDBCDatabaseAndTableTest.java

```diff
@@ -30,93 +41,103 @@
 import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.BeforeClass;
-
-import javax.sql.DataSource;
-import java.io.InputStreamReader;
-import java.sql.SQLException;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
+import org.yaml.snakeyaml.Yaml;
 
 public abstract class AbstractEncryptJDBCDatabaseAndTableTest extends AbstractSQLTest {
-    
+
```

Comment:
should keep 4 blankspace

Reply:
sorry , can't get you . I used project check-style config.

Reply:
plz let origin style here with 4 blankspaces

Reply:
Used same checkstyle. Do you any difference.